### PR TITLE
PMM-2486: Add parameter "--disable-ssl" on exporter.

### PR DESCRIPTION
**Pull request by gywndi**
Add new parameter to disable ssl on expoter.

ex)
* pmm-admin add mysql --disable-ssl
* pmm-admin add mysql:metrics --disable-ssl
* pmm-admin add linux:metrics --disable-ssl
===============================

Please change prometheus.yml, as below. (This must be)
* regex : scheme_https =>  regex: .*,scheme_([\w]+),.* 
* replacement : https => $1 
===============================
- source_labels: [__meta_consul_tags] 
separator: ; 
regex: .*,scheme_([\w]+),.* 
target_label: __scheme__ 
replacement: $1 
action: replace

### CODE REVIEW

File: pmm-admin.go [hunk 1]
```diff
@@ -1216,7 +1216,7 @@ despite PMM server is alive or not.
 	flagMongoURI, flagCluster, flagDSN, flagFormat string
 	flagATags                                      string
 
-	flagVersion, flagJson, flagAll, flagForce bool
+	flagVersion, flagJson, flagAll, flagForce, flagDisableSSL bool
```
Comment (golangci) [inline pmm-admin.go]: var flagJson should be flagJSON

Comment (gywndi) [reply pmm-admin.go]: So placed flagDisableSSL variable only after "flagForce".
Any problem? :-)

Comment (gywndi) [reply pmm-admin.go]: I've got it.
You only say about the variable name.
But it's same as "flagMongoURI" parameter above.
And parameter name is "--disable-ssl", so this name is better. :-)
Thanks.

**Comment by gywndi** [comment]
Do I have to do anything more to merge this code?

**Comment by arvenil** [comment]
Hi @gywndi, thanks for your amazing contribution. I've made changes to prometheus.yml, you can verify them here https://github.com/percona/pmm-server/pull/73

**Comment by gywndi** [comment]
Thanks! :-)

**Comment by rnovikovP** [comment]
@ushuz yes, this is the way. for now, we don't have the way to "update" parameters for client-server communication

**Pull request merged**